### PR TITLE
Setup 2021-04-20 challenge

### PR DESCRIPTION
**Pull request by ericwburden**
Created challenge setup for Potential Palindrome, for 2021-04-20 meetup

### CODE REVIEW

**Review by parhaml — APPROVED** [review]
I like it, will be fun to see the results.

**Comment by ericwburden** [comment]
I can almost guarantee that XOR is not (necessarily) the answer here...

File: challenges/2021-04-20-potential-palindrome/challenge.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+```
+
+The string `carrace` can be rearranged to make `racecar`, a palindrome.
+
+## Tackling This Challenge
+
+1. Make sure you've got the required software on your machine: A JDK 11+
+1. If you haven't already, fork the CodingDojo repository ([INSTRUCTIONS](https://docs.github.com/en/github/getting-started-with-github/fork-a-repo)).
+1. Checkout a new branch using `git checkout -b yourgithubusername-wip`.
+1. Add your code to the 'mob.java' file to make the `countWordsBetween` function work as expected.
```
Comment (birenderjit) [inline challenges/2021-04-20-potential-palindrome/challenge.md:24]: I am guessing this needes to be canMakePalindrome instead of countWordsBetween

Comment (ericwburden) [reply challenges/2021-04-20-potential-palindrome/challenge.md]: Yep.

**Comment by ericwburden** [comment]
Fixed the typo @birenderjit  noted.

**Review by birenderjit — DISMISSED** [review]

**Pull request merged**